scripts/quarantine.yaml

```diff
@@ -7,13 +7,6 @@
 #  platforms:
 #    - all
 
-- scenarios:
```

Comment:
Was this intentional? 
If so, I think it should be a separate commit

Reply:
Taking the tests out of quarantine is intentional. This PR fixes the last issue that caused these tests to fail. I can move it to a separate commit.